### PR TITLE
Update README with new syntax

### PR DESCRIPTION
**Pull request by mrbrendanw**

### CODE REVIEW

**Comment by mrbrendanwong** [comment]
Please add new things you think we should mention into the README

File: README.md [hunk 1]
```diff
@@ -1,29 +1,52 @@
...
 ## Technical Features
-* Constrained image positioning: position your image on screen left, right, top, bottom, center
+* Constrained image positioning: Layer and position your image on screen left, right, top, bottom, center
 * BGM and sound effects: shape the atmosphere with unique background music and sound effects for every screen
 
-## Code Example
+## Grammar
+```
+PROGRAM 		::= GAME_SETTINGS START_SCENE STORY_SCENES DEATH_SCENES END_SCENES
+GAME_SETTINGS	::= "GAME:" "title:" STRING "description:" STRING STATS
```
Comment (BoredX) [inline README.md]: Does this mean the title is always before description, which is always before STATS?

Comment (mrbrendanwong) [reply README.md]: We didn't know how to write it. Was hoping you'd know

File: README.md [hunk 2]
```diff
@@ -1,29 +1,52 @@
...
+STATS			::= "stats" STAT*
+STAT			::= NAME [INTEGER|STRING]
+START_SCENE     ::= "START SCENE:" NAME CHOICE* [TEXT|BGM|SOUND_EFFECT|PICTURE]*
```
Comment (BoredX) [inline README.md]: [TEXT|BGM...]* can use ? to mean at most 1. But we don't handle that in the code anyways, so idk if we should change the grammar for that

Same with the other [..|..]* ones

File: README.md (hunk 1 above)
Comment (BoredX) [reply README.md]: I was thinking of putting CHOICE* inside the [ ]

**Pull request merged**